spec.go

```diff
@@ -438,6 +438,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
 	if err != nil {
 		return nil, err
 	}
+	if spec.Linux.CgroupsPath != "" {
+		name = spec.Linux.CgroupsPath
+	}
```

Comment:
I understand now, thanks for explanation. However, this name can also be a
path, since it gets appended to the cgroup mount.
On Nov 18, 2015 12:16 AM, "Qiang Huang" <notifications@github.com> wrote:

> In spec.go
> <https://github.com/opencontainers/runc/pull/399#discussion_r45170849>:
>
> > @@ -438,6 +438,9 @@ func createCgroupConfig(name string, spec *specs.LinuxRuntimeSpec, devices []*co
> >  	if err != nil {
> >  		return nil, err
> >  	}
> > +	if spec.Linux.CgroupsPath != "" {
> > +		name = spec.Linux.CgroupsPath
> > +	}
>
> Sorry for the confusing, here all cgroup path I mentioned are relative to
> cgroup mount point, and full path I meat absolute path including cgroup
> name, like /web/apache/apache1, here I think what name should be is
> apache1, not the entire /web/apache/apache1.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/399/files#r45170849>.
>